src/vocgui/admins/training_set_admin.py

```diff
@@ -102,7 +104,14 @@ def get_form(self, request, obj=None, **kwargs):
                 .order_by("level")
             )
             form.base_fields["documents"].queryset = Document.objects.filter(
-                created_by__in=[group.name for group in request.user.groups.all()]
+                Q(created_by__in=[group.name for group in request.user.groups.all()])
+                | Q(
+                    id__in=[
+                        obj.id
+                        for obj in Document.objects.filter(creator_is_admin=True)
+                        if obj.is_valid()
+                    ]
```

Comment:
```suggestion
                Q(created_by__in=request.user.groups.all())
                | (
                    Q(creator_is_admin=True, document_image__confirmed=True)
                    & ~Q(audio="")
```
I think this can be simplified, probably there is no need for a python-level `is_valid()` method, because the constraints can be filtered on the database layer.